contracts/round/RoundImplementation.sol

```diff
@@ -128,9 +128,6 @@ contract RoundImplementation is AccessControlEnumerable, Initializable {
   /// @notice MetaPtr to the application form schema
   MetaPtr public applicationMetaPtr;
 
-  /// @notice MetaPtr to the projects
-  MetaPtr public projectsMetaPtr;
```

Comment:
since this is an upgradeable contract, we can't remove variables that change the storage of the proxy